proxy/Main.cc

```diff
@@ -908,7 +908,7 @@ check_fd_limit()
   if (fds_throttle > fds_limit + THROTTLE_FD_HEADROOM) {
     int new_fds_throttle = fds_limit - THROTTLE_FD_HEADROOM;
     if (new_fds_throttle < 1) {
-      MachineFatal("too few file descritors (%d) available", fds_limit);
+      ink_abort("too few file descritors (%d) available", fds_limit);
```

Comment:
Oh I should use ink_abort here since Fatal does not call abort().  I think the original idea was to  terminate immediately at this point without memory cleanup